components/brave_news/browser/resources/shared/Context.tsx

```diff
@@ -127,6 +131,15 @@ export function BraveNewsContextProvider(props: { children: React.ReactNode }) {
     return () => { publishersCache.removeListener(handler) }
   }, [])
 
+  React.useEffect(() => {
+    if (shouldRenderImages) {
+      return
+    }
```

Comment:
sorry, probably don't need this anymore now its been removed from the deps array

Reply:
done.